src/components/OverviewComponent/Carousel.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectProduct, selectProductStyle, selectDefaultProductStyle } from '../../reducers/overviewReducers';
+import Slider from 'react-slick';
+import Zoom from 'react-img-zoom';
+import { ExpandOutlined } from '@ant-design/icons';
+
+
+
+const Carousel = (props) => {
+
+  const dispatch = useDispatch();
+  const product = useSelector(selectProduct);
+  const productStyleList = useSelector(selectProductStyle);
+  const defaultProductStyle = useSelector(selectDefaultProductStyle);
+
+  // carousel component state --->  not shared state
+  const [expanded, setExpanded] = useState(false);
+
+  // expands image when expand icon is clicked on a top right corner of a picture
+  const toggleExpand = () => {
+    expanded ? setExpanded(false) : setExpanded(true);
+  };
+  // settings for photo carousel
+  const settings = {
+    dots: false,
+    variableWidth: true,
+    centerMode: true,
+    infinite: false,
+    speed: 500,
+    initialSlide: 0,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+
+  return (
+    <>
```

Comment:
Are the fragments necessary? (I don't actually know) Doesn't the 'carousel-place-holder' div already contain everything?

Reply:
yes